core/src/commonMain/kotlin/State.kt

```diff
@@ -104,7 +104,42 @@ public sealed class State {
             public object EncryptionTimedOut : Status()
 
             /** Catch-all for any statuses that are unknown for a platform. */
-            public data class Unknown(val status: Int) : Status()
+            public data class Unknown(val status: Int) : Status() {
+                override fun toString(): String {
+                    return status.toString()
+                }
+            }
+
+            override fun toString(): String {
+                return when (this) {
+                    PeripheralDisconnected -> "Peripheral Disconnected"
+                    CentralDisconnected -> "Central Disconnected"
+                    Failed -> "Failed"
+                    L2CapFailure -> "L2Cap Failure"
+                    Timeout -> "Timeout"
+                    LinkManagerProtocolTimeout -> "LinkManager Protocol Timeout"
+                    UnknownDevice -> "Unknown Device"
+                    Cancelled -> "Cancelled"
+                    ConnectionLimitReached -> "Connection Limit Reached"
+                    EncryptionTimedOut -> "Encryption Timed Out"
+                    is Unknown -> "" // check override
+                }
+            }
+        }
+
+        override fun toString(): String {
+            return "Disconnected(${status.toString()})"
+        }
+    }
+
+    override fun toString(): String {
+        return when (this) {
+            Connected -> "Connected"
+            Connecting.Bluetooth -> "Connecting.Bluetooth"
+            Connecting.Observes -> "Connecting.Observes"
+            Connecting.Services -> "Connecting.Services"
+            Disconnecting -> "Disconnecting"
+            is Disconnected -> "" // check override
```

Comment:
Should either `error` or have a valid string. If a future change causes us to actually hit this item, then it would be bad to provide an empty string.

What does "check override" comment mean?